pom.xml

```diff
@@ -231,37 +231,47 @@
             <dependency>
                 <groupId>com.github.cafapi.ssl</groupId>
                 <artifactId>caf-ssl-dropwizard</artifactId>
-                <version>1.0.0-13</version>
+                <version>1.1.0-56</version>
             </dependency>
             <dependency>
                 <groupId>com.github.cafapi.util</groupId>
                 <artifactId>util-moduleloader</artifactId>
-                <version>3.1.0-596</version>
+                <version>3.2.0-684</version>
             </dependency>
             <dependency>
                 <groupId>com.github.cafapi.util</groupId>
                 <artifactId>util-naming</artifactId>
-                <version>3.1.0-596</version>
+                <version>3.2.0-684</version>
             </dependency>
             <dependency>
                 <groupId>com.github.cafapi.util</groupId>
                 <artifactId>util-process-identifier</artifactId>
-                <version>3.1.0-596</version>
+                <version>3.2.0-684</version>
             </dependency>
             <dependency>
                 <groupId>com.github.cafapi.util</groupId>
                 <artifactId>util-ref</artifactId>
-                <version>3.1.0-596</version>
+                <version>3.2.0-684</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.cafapi.util</groupId>
+                <artifactId>util-secret</artifactId>
+                <version>3.2.0-684</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.cafapi.util.dropwizard</groupId>
+                <artifactId>caf-substitutor-dropwizard</artifactId>
+                <version>1.0.0-7</version>
             </dependency>
```

Comment:
@rorytorneymf I notice that this `caf-substitutor-dropwizard` dependency was removed by the dependency updating utility, which suggests to me that it's not used.  Does that sound right?

Reply:
@dermot-hardy Thanks, its not actually used currently as we don't have any passwords in the dropwizard config yaml,  but should still be configured to be available in future, added it here:

https://github.com/JobService/job-service/pull/456/files